yarn.lock

```diff
@@ -12,9 +12,9 @@
     js-yaml "^3.13.1"
 
 "@apollo/protobufjs@^1.0.3":
```

Comment:
Let's avoid a full `yarn.lock` refresh here. Please reset to the one in master and just include the ones needed by the plugin.

Reply:
Absolutely!